lib/uuid/Kconfig

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2024, SECO Mind Srl
+#
+# SPDX-License-Identifier: Apache-2.0
+
+menu "Universally Unique Identifier (UUID)"
+
+config UUID
+	bool "UUID support [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	help
+	  Enable use of the UUID library.
```

Comment:
I realize that this comment is a bit last, but do we even need this at all? 

If none of the below is enabled, and none of the functions are called, the linker should remove the entire file by itself anyways.

Reply:
Removing this would make always available to the user functions such as `uuid_copy`, `uuid_from_buffer` and `uuid_from_string`. Since this is an experimental module I think it makes more sense to provide those functions only when explicitly using experimental features.

Reply:
Sure. Since it's experimental, we could remove it when we it stops being experimental